gateway/src/test/kotlin/helper/LinearRetryTest.kt

```diff
@@ -0,0 +1,24 @@
+package helper
+
+import dev.kord.gateway.retry.LinearRetry
+import kotlinx.coroutines.runBlocking
+import kotlin.test.Test
+import kotlin.time.Duration.Companion.milliseconds
+
+class LinearRetryTest {
+
+    @Test
+    fun testLinearity() {
```

Comment:
Could you maybe also add tests for extreme cases you described in the issue? (like `1`, `Int.MAX_VALUE`, ...).
Also to make these test independent of real time, the delay-skipping `runTest {}` can be used with `currentTime` for getting the current virtual time.

Reply:
oh thanks for the tip I didn't know that existed